frontend/packages/console-dynamic-plugin-sdk/src/extensions/console-types.ts

```diff
@@ -734,6 +734,12 @@ export type UseValuesForNamespaceContext = () => {
 
 export type UseActiveNamespace = () => [string, (ns: string) => void];
 
+export type UseUserSettings = <T>(
```

Comment:
This type is also defined in `packages/console-dynamic-plugin-sdk/src/api/internal-types.ts`

Can we remove that one in favor of this one?

Reply:
Good catch.  That was an oversight.  I thought I'd removed it.